src/Middleware/Diagnostics/src/DeveloperExceptionPage/DeveloperExceptionPageMiddleware.cs

```diff
@@ -63,6 +66,13 @@ public class DeveloperExceptionPageMiddleware
             _fileProvider = _options.FileProvider ?? hostingEnvironment.ContentRootFileProvider;
             _diagnosticSource = diagnosticSource;
             _exceptionDetailsProvider = new ExceptionDetailsProvider(_fileProvider, _options.SourceCodeLineCount);
+            _exceptionHandler = DisplayException;
+
+            foreach (var filter in filters.Reverse())
```

Comment:
needs a null check on filtars

Reply:
In case somebody news this up manually?

Reply:
other params has it. I'm not going to make a fuss, I mentioned it in case you just plum forgot. I tend to do this these days because I don't want people writing unit tests that partially pass in stuff.

Reply:
I added it. But I wasn't sure if we were unit testing `IE<T>` because we think people will pass null manually or something else.